homeassistant/components/homekit_controller/climate.py

```diff
@@ -219,6 +257,8 @@ async def async_set_hvac_mode(self, hvac_mode: HVACMode) -> None:
     def target_temperature(self) -> float | None:
```

Comment:
This function should return None when in heat cool mode.

Reply:
fixed